dali/kernels/test/tensor_test_utils.h

```diff
@@ -65,28 +62,36 @@ void CheckEqual(
   }
 }
 
-template <int dim1, int dim2>
+
+template<int dim1, int dim2>
 void CheckEqual(const TensorListShape<dim1> &s1, const TensorListShape<dim2> &s2) {
   int max_errors = detail::DefaultMaxErrors;
   CheckEqual(s1, s2, max_errors);
 }
 
-template <typename T>
+
+template<typename T>
```

Comment:
Why do you remove the space between `template` and `<`? It's counterproductive, we have .clang-format that adds the spaces automatically  and this just makes the diffs bigger.

Reply:
My bad. Missed it in my autoformat conf